core/components/consent-manager/consent-providers.tsx

```diff
@@ -21,15 +20,13 @@ export function ConsentManagerProvider({
   return (
     <C15TConsentManagerProvider
       options={{
-        mode: 'custom',
-        consentCategories: ['necessary', 'functionality', 'marketing', 'measurement'],
-
-        // @ts-expect-error endpointHandlers type is not yet exposed by the package
-        endpointHandlers: {
-          showConsentBanner: () => showConsentBanner(isCookieConsentEnabled),
-          setConsent,
-          verifyConsent,
+        mode: 'offline',
+        storageConfig: {
+          storageKey: CONSENT_COOKIE_NAME,
+          crossSubdomain: true,
```

Comment:
Enabling cross-subdomain doesn't interfere with consent cookie names that may be set on the hosted checkout, does it?

Reply:
Per an offline conversation, it could conflict but cookie names are different so it should be fine.